<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>271</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="convertToGray">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
     </property>
     <property name="text">
      <string>Convert to Gray</string>
     </property>
    </widget>
    <widget class="QPushButton" name="negative">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
     </property>
     <property name="text">
      <string>Negative</string>
     </property>
    </widget>
    <widget class="QLabel" name="brightness_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 20pt &quot;Futura&quot;;</string>
     </property>
     <property name="text">
      <string>Brightness</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_brightness">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="brightness_label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>270</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 20pt &quot;Futura&quot;;</string>
     </property>
     <property name="text">
      <string>Contrast</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_contrast">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="grayQuantization_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 16pt &quot;Futura&quot;;</string>
     </property>
     <property name="text">
      <string> Gray Quantization</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>341</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>COPY </string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="edited_label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>670</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Edited</string>
    </property>
   </widget>
   <widget class="QLabel" name="canvas_edited">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>370</y>
      <width>311</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="original_label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>670</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
   </widget>
   <widget class="QLabel" name="canvas_original">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>311</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>260</y>
      <width>261</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="Luminance">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
     </property>
     <property name="text">
      <string>Luminance</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Lab">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
     </property>
     <property name="text">
      <string>L*a*b</string>
     </property>
    </widget>
    <widget class="QPushButton" name="equalization">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>121</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
     </property>
     <property name="text">
      <string>Equalization</string>
     </property>
    </widget>
    <widget class="QPushButton" name="matching">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 20pt &quot;Futura&quot;;</string>
     </property>
     <property name="text">
      <string>Matching</string>
     </property>
    </widget>
    <widget class="QLabel" name="convolution_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 20pt &quot;Futura&quot;;</string>
     </property>
     <property name="text">
      <string>Convolution</string>
     </property>
    </widget>
    <widget class="QComboBox" name="convolutionFilters">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="filtersTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>201</width>
       <height>101</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>3</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>66</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>33</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QPushButton" name="applyFilter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>221</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="mais127">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>360</y>
       <width>86</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>+127</string>
     </property>
    </widget>
   </widget>
   <widget class="QChartView" name="histogram_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1221</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Flip_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 20pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Mirroring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flip_Vertically">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Vertically</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flip_Horizontally">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Horizontally</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotate_label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>270</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 20pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rot_Clockwise">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Clockwise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rot_Counterclockwise">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Counterclockwise</string>
    </property>
   </widget>
   <widget class="QLabel" name="zoom_label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>280</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 20pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Zoom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomIn">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>300</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Zoom in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomOut">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>330</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 15pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Zoom out</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1234</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionOpen_Image"/>
    <addaction name="actionSave_Image"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionOpen_Image">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/icons/img/open_folder.png</normaloff>:/new/icons/img/open_folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/icons/img/save_image.png</normaloff>:/new/icons/img/save_image.png</iconset>
   </property>
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
